### PR TITLE
PR_SOLVING_ISSUE_#9

### PR DESCRIPTION
**Pull request by caemadmin**
/solves https://github.com/caemate/generic-target/issues/9
Pull SOLVING_ISSUE_#9

### CODE REVIEW

File: file-issue-nine.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+console.log('issue nine');
+
+
+console.log('test 1');
```
Comment (caemadmin) [inline file-issue-nine.js:4]: comment_1

File: file-issue-nine.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+console.log('test 2');
```
Comment (caemadmin) [inline file-issue-nine.js:5]: comment_2

File: file-issue-nine.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+console.log('test 7');
+console.log('test 9');
+console.log('test 6');
```
Comment (caemadmin) [inline file-issue-nine.js:8]: REVIEW_4

Comment (caemadmin) [reply file-issue-nine.js:8]: REVIEW_4 comment#2

**Review by caemadmin — COMMENTED** [review]
Update PULL_REVIEW body content UPDATE

**Comment by caemadmin** [comment]
test

**Review by caemadmin — COMMENTED** [review]
REVIEW COMMENT

**Review by devcaem — APPROVED** [review]

**Review by devcaem — APPROVED** [review]

**Review by devcaem — APPROVED** [review]

**Review by devcaem — APPROVED** [review]

**Review by devcaem — APPROVED** [review]

**Review by devcaem — APPROVED** [review]

**Review by devcaem — APPROVED** [review]

**Review by devcaem — APPROVED** [review]

**Review by devcaem — APPROVED** [review]

**Review by devcaem — APPROVED** [review]

**Review by devcaem — APPROVED** [review]

**Review by devcaem — APPROVED** [review]

**Review by devcaem — APPROVED** [review]

**Review by devcaem — APPROVED** [review]

**Review by devcaem — APPROVED** [review]

**Review by devcaem — APPROVED** [review]

**Review by devcaem — APPROVED** [review]

**Review by devcaem — APPROVED** [review]

File: file-issue-nine.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+console.log('test 6');
+console.log('test 4');
+console.log('test 3');
```
Comment (devcaem) [inline file-issue-nine.js:11]: For the reviewer from the developer.
Is this correct?

Comment (devcaem) [reply file-issue-nine.js:11]: In need this details ..... ?
From developer

Comment (devcaem) [reply file-issue-nine.js:11]: I need this other things
- question 1
- question 2
- question 3

**Review by devcaem — COMMENTED** [review]
Review COMMENT, by developer

**Review by devcaem — APPROVED** [review]

**Review by devcaem — APPROVED** [review]

**Review by devcaem — APPROVED** [review]

**Review by devcaem — APPROVED** [review]

**Review by devcaem — APPROVED** [review]

**Review by devcaem — APPROVED** [review]

**Review by devcaem — APPROVED** [review]

**Review by devcaem — APPROVED** [review]

**Review by devcaem — APPROVED** [review]

**Review by devcaem — APPROVED** [review]

**Review by devcaem — APPROVED** [review]

**Review by devcaem — APPROVED** [review]

**Review by revcaem — APPROVED** [review]

File: file-issue-nine.js [hunk 5]
```diff
@@ -0,0 +1,16 @@
+console.log('issue nine');
+
+
+console.log('test 1');
+console.log('test 2');
+console.log('test 7');
+console.log('test 9');
+console.log('test 6');
+console.log('test 6');
+console.log('test 4');
```
Comment (revcaem) [inline file-issue-nine.js:10]: Rev review comment

File: file-issue-nine.js [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+console.log('test 3');
+console.log('test 3');
+
+
```
Comment (revcaem) [inline file-issue-nine.js:14]: Rev second review comment

**Review by revcaem — APPROVED** [review]

**Review by revcaem — APPROVED** [review]

**Review by devcaem — APPROVED** [review]

**Review by revcaem — APPROVED** [review]

**Review by devcaem — APPROVED** [review]

**Review by revcaem — APPROVED** [review]

**Review by devcaem — APPROVED** [review]

File: file-issue-nine.js [hunk 7]
```diff
@@ -0,0 +1,16 @@
+console.log('issue nine');
+
+
+console.log('test 1');
+console.log('test 2');
+console.log('test 7');
+console.log('test 9');
+console.log('test 6');
+console.log('test 6');
```
Comment (devcaem) [inline file-issue-nine.js:9]: Dev comment thread 1 - comment 1

Comment (devcaem) [reply file-issue-nine.js:9]: Dev comment thread 1 - com2

**Review by devcaem — APPROVED** [review]

File: file-issue-nine.js (hunk 3 above)
Comment (devcaem) [reply file-issue-nine.js:8]: reply

Comment (devcaem) [reply file-issue-nine.js:8]: reply 1

**Comment by caemadmin** [comment]
comment_on_pull_21

**Pull request closed without merging**